### ISSUE TITLE
several errors

### ISSUE THREAD

**Issue by flybyray** [janmotl/linkifier#2]
The java class "example.Linkifier" is missing from github project

Testing with a mssql server (docker mssql https://github.com/Microsoft/mssql-docker | https://hub.docker.com/r/microsoft/mssql-server-linux/ )
got something like this

Successfully connected to the database.
Table count: 443
Cannot resolve the collation conflict between "Latin1_General_CI_AS" and "SQL_Latin1_General_CP1_CI_AS" in the equal to operation.
Something went wrong...

Contact the developer at **** => the printed email is not working

**Comment by flybyray** [comment]
Ok fixed that. was really a problem with mssql linux docker
https://github.com/Microsoft/mssql-docker/issues/6

But know it fails with
```
Successfully connected to the database.
Table count: 443
Column count: 11413
null
Something went wrong...
Contact the developer at jan.motl@fit.cvut.cz

```
now it must be debugged but i dont know where to get the missing classes to load the project in an IDE environment

**Comment by janmotl** [comment]
Thank you for the report.

1. I uploaded the missing "example.Linkifier" class. My fault.
2. For faster debugging, it may be better to bypass the GUI and just modify "example.Demo" class.
3. Can you try the email again? It seems to be working correctly. But if problems persist, I will replace it with a gmail email.

**Comment by flybyray** [comment]
ok i found a nullpointer.
maybe a fix would be this:
https://gist.github.com/flybyray/79ca292674185cbbe317757edb2b5ee0
worked but i dont know if that is correct to set a ratio to '0' here

**Comment by janmotl** [comment]
Thanks. The problem should be now fixed in Linkifier v3.1.2.

**Comment by flybyray** [comment]
here you are! and thanks too! just tried to get some structure for hibernate mapping from big legacy mssql database.
Linkifier is little, nice and with good results.
But unfortunately in my case only partly helpful, because of messy blown database design.
